tasks/cat3.yml

```diff
@@ -1,4 +1,893 @@
+  always_run: yes
+  tags:
+      - cat3
+      - low
+      - V-38454
+      - rpm
+      - audit
+
+- name: "| LOW | V-38454 | PATCH |\n
+        The system package management tool must verify ownership on all files and directories associated with packages."
+  command: rpm --setugids {{ item.stdout }}
+  when: item|changed
+  with_items: rpm_group_ownership_audit.results
+  tags:
+      - cat3
+      - low
+      - V-38454
+      - rpm
+      - audit
+
+# V-38455 see not_automated.yml
+
+# V-38456 see not_automated.yml
+
+# V-38460 NFS thing
+
+# V-38463 see not_automated.yml
+
+# V-38467 see not_automated.yml
+
+# V-38471 audispd syslog plugin
+- name: "| LOW | V-38471 | AUDIT |\n
+        The system must forward audit records to the syslog service."
+  command: grep '^active' /etc/audisp/plugins.d/syslog.conf
+  register: auditd_syslog_output_audit
+  failed_when: auditd_syslog_output_audit.rc == 2
+  changed_when: false
+  always_run: yes
+  tags:
+      - cat3
+      - low
+      - V-38471
+      - auditd
+      - audit
+
+- name: "| LOW | V-38471 | PATCH |\n
+        The system must forward audit records to the syslog service."
+  lineinfile:
+      state: present
+      backup: no
+      dest: /etc/audisp/plugins.d/syslog.conf
+      regexp: '^#?active'
+      line: 'active = yes'
+  notify: restart auditd
+  tags:
+      - cat3
+      - low
+      - V-38471
+      - auditd
+      - patch
+
+# V-38473 see not_automated.yml
+
+# V-38474 lock gui
+- name: "| LOW | V-38474 | AUDIT |\n
+        Low  The system must allow locking of graphical desktop sessions."
+  command: gconftool-2 --direct --config-source xml:readwrite:/etc/gconf/gconf.xml.mandatory --get /apps/gnome_settings_daemon/keybindings/screensaver
+  when: rhel6stig_xwindows_required
+  register: gui_screen_lock_hotkey_audit
+  always_run: yes
+  tags:
+      - cat3
+      - low
+      - V-38474
+      - screen_lock
+      - audit
+
+- name: "| LOW | V-38474 | PATCH |\n
+        Low  The system must allow locking of graphical desktop sessions."
+  command: gconftool-2 --direct --config-source xml:readwrite:/etc/gconf/gconf.xml.mandatory --type string --set /apps/gnome_settings_daemon/keybindings/screensaver "<Ctrl><Alt>l"
+  when: rhel6stig_xwindows_required
+  tags:
+      - cat3
+      - low
+      - V-38474
+      - screen_lock
+      - patch
+
+# V-38478 rhnsd service
+- name: "| LOW | V-38478 | AUDIT |\n
+        The Red Hat Network Service (rhnsd) service must not be running, unless using RHN or an RHN Satellite."
+  command: chkconfig 'rhnsd' --list
+  register: rhnsd_service_audit
+  always_run: yes
+  changed_when: false
+  ignore_errors: yes
+  tags:
+      - cat3
+      - low
+      - V-38478
+      - rhnsd
+      - audit
+
+- name: "| LOW | V-38478 | PATCH |\n
+        The Red Hat Network Service (rhnsd) service must not be running, unless using RHN or an RHN Satellite."
+  shell: chkconfig rhnsd off && chkconfig rhnsd --list
+  register: result
```

Comment:
let's try to avoid ambiguous registered vars instead of `result` perhaps `rhsnd_check`